### PR TITLE
activate cccd callback / Add an api for set gap device name

### PR DESCRIPTION
**Pull request by ydlee-sec**
activate cccd callback / Add an api for set gap device name

### CODE REVIEW

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c [hunk 1]
```diff
@@ -389,6 +389,11 @@ trble_conn_handle rtw_ble_server_get_conn_handle_by_address(uint8_t* mac)
     return conn_id;
 }
 
+trble_result_e rtw_ble_server_set_device_name(uint8_t* name)
+{
+    le_set_gap_param(GAP_PARAM_DEVICE_NAME, GAP_DEVICE_NAME_LEN, name);
+    return TRBLE_SUCCESS;
+}
```
Comment (ziyik) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c:396]: Could you correct the spacing here?

Comment (ydlee-sec) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c:396]: How ? Do you mean add a new line after ```line 396```?

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c:396]: Yes, you are right.

**Review by hs36-kim — APPROVED** [review]
LGTM

**Pull request merged**